docs/configurable-imports.md

```diff
@@ -0,0 +1,101 @@
+# Configurable Imports
+
+If you are using ES Module imports (`import styled from 'emotion'`)
```

Comment:
`styled` doesn't exist in emotion. It's only in react-emotion and preact-emotion.